README.md

```diff
@@ -28,7 +28,7 @@ Assumptions:
 
 4. Switch to the alpha release version of the code. This is the latest stable release.
 
-        $ git checkout v0.1.0-alpha
+        $ git checkout staging
```

Comment:
The latest stable release would be `master` as opposed to `staging`. Line 33 down below instructs the user to refer to staging for the latest, work-in-progress features.